spaceship/models/mission.py

```diff
@@ -22,3 +22,24 @@ class Mission(BaseModel):
   @hybrid_property
   def is_active(self):
     return self.deleted_at == None
+
+  @hybrid_property
+  def start_time(self):
```

Comment:
maybe worth calling this "formatted_start_time" or "start_time_str"?

Reply:
yes, good point!